released/packages/coq-mathcomp-bigenough/coq-mathcomp-bigenough.1.0.0/opam

```diff
@@ -13,6 +13,7 @@ install: [ make "install" ]
 remove: [ "sh" "-c" "rm -rf '%{lib}%/coq/user-contrib/mathcomp/bigenough'" ]
 depends: [
   "ocaml"
+  "coq" {((>= "1.6" && < "8.15~") | = "dev")}
```

Comment:
```suggestion
  "coq" {>= "1.6" & < "8.15~"}
```
I don't see the point of having `dev` here, since it for sure doesn't work with `coq.dev`. And the syntax was wrong.

Reply:
The lower bound 1.6 looks more like a mathcomp version number to me?

Reply:
OK, how about we change the lower bound to 8.5?